conda/recipes/cudf/meta.yaml

```diff
@@ -45,6 +45,7 @@ requirements:
     - fsspec>=0.6.0
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
     - nvtx >=0.2.1
+    - packaging
```

Comment:
Does this need to be added to the `integration` repo as well?

Reply:
Would defer to @ajschmidt8 here, there was a build job-related failure at this line https://github.com/rapidsai/cudf/blob/branch-0.19/ci/cpu/build.sh#L78-L79 so AJ suggested we add it here.

Reply:
yup, i will open a PR.

Reply:
https://github.com/rapidsai/integration/pull/225

Feel free to review. I will wait to merge until this PR is confirmed passing also.